site/content/events/2015-boston/_sponsors.txt

```diff
@@ -26,7 +26,11 @@ end
 
 <% @ssponsors = [
 { :image => "conjur.png", :name => "Conjur", :link => "http://www.conjur.net/"},
-{ :image => "puppetlabs.jpg", :name => "Puppet Labs", :link => "http://www.puppetlabs.com"}
+{ :image => "puppetlabs.jpg", :name => "Puppet Labs", :link => "http://www.puppetlabs.com"},
+{ :image => "xebia.png", :name => "Xebialabs", :link => "https://xebialabs.com/"},
+{ :image => "nutanix.png", :name => "Nutanix", :link => "http://www.nutanix.com/"},
+{ :image => "appneta.png", :name => "AppNeta", :link => "http://www.appneta.com/"},
+{ :image => "cimpress.png", :name => "Cimpress", :link => "http://cimpress.com/"}
```

Comment:
Pro Tip: :tm: Use trailing commas for these lists, that way future additions are one-liners (and thus easier to visually scan prior to merge). :smile: